boxoffice/assets/js/models/util.js

```diff
@@ -28,6 +28,17 @@ export const Util = {
     else {
       return moment(dateTimeString).toString();
     }
+  },
+  getElementId: function(htmlString) {
+    return htmlString.match(/id="(.*?)"/)[1];
+  },
+  getFormTemplate: function(htmlString) {
+    // Add on click event handler for Ractive to submit the form
+    const formTemplate = `${htmlString.slice(0, htmlString.search(/type="submit"/))} on-click="onFormSubmit(event)" {{#formOnSubmit}}disabled{{/}} ${htmlString.slice(htmlString.search(/type="submit"/))}`;
+    return formTemplate
+  },
+  showFormErrors: function(formId, errors) {
```

Comment:
Is this function required? Can call the underlying function directly, since this function just passes the call through?

Reply:
Fixed